internal/bminventory/inventory.go

```diff
@@ -3252,6 +3258,10 @@ func (b *bareMetalInventory) setMajorityGroupForCluster(clusterID *strfmt.UUID,
 	return b.clusterApi.SetConnectivityMajorityGroupsForCluster(*clusterID, db)
 }
 
+func (b *bareMetalInventory) detectAndStoreCollidingIPsForCluster(clusterID *strfmt.UUID, db *gorm.DB) error {
```

Comment:
I think we need to call it only from monitor.

Reply:
@ori-amizur what are the possible triggers for change in the collision status?

Reply:
Change in connectivity result received from host